### ISSUE TITLE
The implementation seems a little different from the paper

### ISSUE THREAD

**Issue by YLyeliang** [lxtGH/SFSegNets#28]
Hi, first of all, thanks for sharing your work for open access.
When reviewing the code, I found that the decoding part in the FPN. The paper said that it uses multiple FAM to process feature maps of outputs from each decoding layer, and concatenate results together into 4x stride resolution, then predict their final result.
While in the code, the FAM module only used in the step-by-step upsample in the FPN architecture, without using it to get the final concatenated logits.
I wanna know that does it matter for performance?
For now, I think the result is good, it's a good trade-off between speed and accuracy. Besides, adding more FAM may lower the speed, so if anyone tried, please share your experience.

**Comment by lxtGH** [comment]
In the original implementation. we found adding more FAMs leads to performance gains (0.3%). In the latter version such as in this codebase,  it seem like only 3 FAM can lead to considerable results as long as trainig SFNet longer. To achieves the best speed and accuracy trade-off, I adopt the 3FAM design.
